sale_semaphore/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module adds a commercial semaphore to provide visual feedback on the
+prices being offered. It also adds a widget to allow selection of the
```

Comment:
It should be great to explain in which manner as it seems not clear to me at this point.